ultra/ultra/train.py

```diff
@@ -108,6 +108,8 @@ def train(
     # policy_classes list, transform it to an etag of "dqn-v0:ppo-v0".
     etag = ":".join([policy_class.split(":")[-1] for policy_class in policy_classes])
 
+    episode_count = 0
```

Comment:
I think the episode objects have an `index` attribute that keeps track of the episode number. Is there any reason why a separate `episode_count` variable was used instead of `episode.index`? Both are fine, but just wondering.

Reply:
Missed this, thanks.